modules/trusted-firmware-m/Kconfig

```diff
@@ -0,0 +1,140 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config TFM_MINIMAL
+	bool "Use minimal TFM build"
+	imply TFM_IPC
```

Comment:
in this case, the following might be a better choice:
```
config TFM_IPC
	default y if TFM_MINIMAL

config TFM_MINIMAL
	bool "Use minimal TFM build"
```

as that will still allow someone to overrule the `default y`.

Reply:
will do